programmer-skills/big-O-notation.md

```diff
@@ -0,0 +1,161 @@
+###Big-O notation explained  
```

Comment:
You could add a title (Big-O notation) and then name the first section 'Introduction' (as it's customary to have this in READMEs)